### ISSUE TITLE
No Editor after phpList update

### ISSUE THREAD

**Issue by bluepup** [bramley/phplist-plugin-contentareas#5]
I've updated from 3.2.4 to 3.2.5. After that when I try to edit an Area I only get the source code of the area in a textbox. Not the WYSIWYG editor.
After saving without editing anything I get the message:
`A problem with content area named "contactcontent". DOMDocumentFragment::appendXML(): Entity: line 8: parser error : Premature end of data in tag br line 8`
because it seems that somehow all XHTML `<br />` tags are getting replaces with `<br>`. It happens when loading the content area into the textbox, because the textbox already shows the `<br>`. If I check the template the `<br />` tags are correct.

**Comment by bramley** [comment]
Please use the code button <> for bits of html

**Comment by bluepup** [comment]
Sorry, corrected the code bits.
I'm also not sure if this is an issue with the plugin or if the update of phpList went wrong.

**Comment by bramley** [comment]
Which editor are you using? Is it displayed on the Content tab?

**Comment by bluepup** [comment]
I've investigated.
The CKEditor plugin was enabled. But it didn't work even in templates without contentareas.
So I changed to the FCKeditor plugin and then both work. So it must be an error with the CKEditor plugin, not contentareas plugin. Sorry for being too hasty with opening this issue.
Thanks for your immediate response. Great plugin!

**Comment by bramley** [comment]
CKEditor is mine as well so I am interested in finding out why it is not working. If you get the time can you use Firefox web console (or similar in other browsers) to see whether there are any request failures when you load the editor on the Content tab.

There is a change to that plugin in phplist 3.2.5 https://www.phplist.org/newslist/phplist-3-2-5-whats-new/ but there hasn't been any other problems reported.

**Comment by bluepup** [comment]
Looked into it with firebug:

Network shows
`GET http://cdn.ckeditor.com/4.5.7/full/ckeditor.js?_=1466110505605 200 OK cdn.ckeditor.com 163,1 KB`
Console shows:

```
ReferenceError: CKEDITOR is not defined
https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js line 2 > eval
Line 2
```

The code is:

```
CKEDITOR.replace(
                'content',
                {
                    allowedContent: true,
filebrowserBrowseUrl: 'plugins/CKEditorPlugin/kcfinder/browse.php?opener=ckeditor&type=files',
filebrowserImageBrowseUrl: 'plugins/CKEditorPlugin/kcfinder/browse.php?opener=ckeditor&type=image',
filebrowserFlashBrowseUrl: 'plugins/CKEditorPlugin/kcfinder/browse.php?opener=ckeditor&type=flash',
filebrowserUploadUrl: 'plugins/CKEditorPlugin/kcfinder/upload.php?opener=ckeditor&type=files',
filebrowserImageUploadUrl: 'plugins/CKEditorPlugin/kcfinder/upload.php?opener=ckeditor&type=image',
filebrowserFlashUploadUrl: 'plugins/CKEditorPlugin/kcfinder/upload.php?opener=ckeditor&type=flash',
height: 400
                }
            );
```

**Comment by bluepup** [comment]
Don't know exactly what it was, but it works now.
After the update when it didn't work, i tried to change the "URL of ckeditor.js" from
`//cdn.ckeditor.com/4.5.7/full/ckeditor.js` to `http://cdn.ckeditor.com/4.5.7/full/ckeditor.js`
but that didn't work either.
Now after I noticed that Fckeditor worked and I wanted to help finding the bug I changed back to CKEditor and noticed that there was a space in front of the "URL of ckeditor.js".
Now with the space removed and `//cdn.ckeditor.com/4.5.7/full/ckeditor.js` all works.
